modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -151,6 +151,242 @@ GAPI_FLUID_KERNEL(GFluidAddW, cv::gapi::core::GAddW, false)
 
 enum Arithm { ARITHM_ABSDIFF, ARITHM_ADD, ARITHM_SUBTRACT, ARITHM_MULTIPLY, ARITHM_DIVIDE };
 
+#if CV_SIMD
+template<typename T, typename VT>
+static inline void absdiff_impl(const T in1[], const T in2[], T out[], int length, int& x)
```

Comment:
> void ... int& x

It is better to "return x" instead (avoid code which may block compiler optimizations).

Reply:
Done